<launch>
  <rosparam command="load" file="$(find master_krai)/config/var_gajah.yaml"/>  
  
  <arg name="in_joy" default="/dev/input/js0"/>
  <arg name="conf_ps4" default="$(find master_krai)/config/jetson_ps4.yaml"/>
  <arg name="stm" default="/dev/ttyACM0"/>  

  <include file="$(find joy_2)/launch/ps4joy.launch">
    <arg name="dev" value="$(arg in_joy)"/>
    <arg name="conf" value="$(arg conf_ps4)"/>
    <arg name="trig_val" value="false"/>
  </include>
  
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen" respawn="true">    
    <param name="port" value="$(arg stm)"/>
    <param name="baud" value="115200"/>
  </node>

  <node name="example_node" pkg="master_krai" type="example.py" output="screen" respawn="true">
  </node>
         
  <node name="exkinematics_node" pkg="master_krai" type="ex_kinematics.py" output="screen" respawn="true">
  </node>

</launch>
